source/tutorials/fmt-cdiff.rst

```diff
@@ -0,0 +1,478 @@
+
+* Beta diversity
+
+  * Jaccard distance (a qualitative measure of community dissimilarity)
+  * Bray-Curtis distance (a quantitative measure of community dissimilarity)
+  * unweighted UniFrac distance (a qualitative measure of community dissimilarity that incorporates phylogenetic relationships between the features)
+  * weighted UniFrac distance (a quantitative measure of community dissimilarity that incorporates phylogenetic relationships between the features)
+
+An important parameter that needs to be provided to this script is ``--p-sampling-depth``, which is the even sampling (i.e. rarefaction) depth. Because most diversity metrics are sensitive to different sampling depths across different samples, this script will randomly subsample the counts from each sample to the value provided for this parameter. For example, if you provide ``--p-sampling-depth 500``, this step will subsample the counts in each sample without replacement so that each sample in the resulting table has a total count of 500. If the total count for any sample(s) are smaller than this value, those samples will be dropped from the diversity analysis. Choosing this value is tricky. We recommend making your choice by reviewing the information presented in the ``table.qzv`` file that was created above and choosing a value that is as high as possible (so you retain more sequences per sample) while excluding as few samples as possible.
+
+.. question::
+   View the ``table.qzv`` QIIME 2 artifact, and in particular the *Interactive Sample Detail* tab in that visualization. What value would you choose to pass for ``--p-sampling-depth``? How many samples will be excluded from your analysis based on this choice? How many total sequences will you be analyzing in the ``core-metrics-phylogenetic`` command?
+
+.. command-block::
+
+   qiime diversity core-metrics-phylogenetic \
+     --i-phylogeny rooted-tree.qza \
+     --i-table table.qza \
+     --p-sampling-depth 1000 \
+     --m-metadata-file sample-metadata.tsv \
+     --output-dir core-metrics-results
+
+Here we set the ``--p-sampling-depth`` parameter to 1109. This value was chosen based on the number of sequences in the ``L3S341`` sample because it's close to the number of sequences in the next few samples that have higher sequence counts, and because it is considerably higher (relatively) than the number of sequences in the one sample that has fewer sequences. This will allow us to retain most of our samples. The one sample that has fewer sequences will be dropped from the ``core-metrics-phylogenetic`` analyses and anything that uses these results.
+
+.. note:: The sampling depth of 1109 was chosen based on the DADA2 feature table summary. If you are using a Deblur feature table rather than a DADA2 feature table, you might want to choose a different even sampling depth. Apply the logic from the previous paragraph to help you choose an even sampling depth.
```

Comment:
the files produced in the online docs will be deblur by default (since the `mv` command will overwrite the `table.qza` from dada2), so I recommend either switching the order or base rarefaction decisions on the deblur results.

Reply:
@nbokulich, I don't think that is actually the case here:

https://github.com/qiime2/docs/pull/353/files#diff-7ea23c0c78f0ea4acac01869cf88666aR190